src/app/backend/resource/common/resourcechannels.go

```diff
@@ -22,7 +22,7 @@ import (
 	batch "k8s.io/api/batch/v1"
 	batch2 "k8s.io/api/batch/v1beta1"
 	v1 "k8s.io/api/core/v1"
-	extensions "k8s.io/api/extensions/v1beta1"
+	networkingv1 "k8s.io/api/networking/v1"
```

Comment:
I'd recommend doing the migration from `extensions/v1beta1` to specific stable packages in a separate PR for clarity. I see that it will be removed in the K8S `v1.22`

https://kubernetes.io/docs/reference/using-api/deprecation-guide/#ingress-v122